README.md

```diff
@@ -48,14 +48,16 @@ limitations under the License.
 </h4>
 
 <h3 align="center">
-    <p>State-of-the-art Natural Language Processing for Jax, PyTorch and TensorFlow</p>
+    <p>State-of-the-art Machine Learning for Jax, PyTorch and TensorFlow</p>
 </h3>
 
 <h3 align="center">
     <a href="https://hf.co/course"><img src="https://raw.githubusercontent.com/huggingface/transformers/master/docs/source/imgs/course_banner.png"></a>
 </h3>
 
-🤗 Transformers provides thousands of pretrained models to perform tasks on texts such as classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. Its aim is to make cutting-edge NLP easier to use for everyone.
+🤗 Transformers provides thousands of pretrained models to perform tasks on different modalities such as Natural Language Processing, Computer Vision, and Audio. 
```

Comment:
NLP is not really a modality (more like a domain in AI), I would either say "on different modalities such as language, vision and audio", or say "perform tasks on various domains of AI such as Natural Language Processing, Computer Vision, Audio.

And maybe also mention that models might involve several modalities at the same time.